toastify/toastify.nuspec

```diff
@@ -4,9 +4,9 @@
   <metadata>
    <id>toastify</id>
    <title>toastify</title>
-   <version>1.7.8</version>
+   <version>1.8.3</version>
    <authors>Geoffrey Huntley</authors>
-   <owners>ghuntley</owners>
+   <owners>ghuntley, slayerice09</owners>
```

Comment:
Where do we stand with this? Are you still maintaining this? If so, I'll remove my name from the file and let you update it. If not, I'm happy to take over maintaining toastify...